iron-list.html

```diff
@@ -1028,7 +1028,7 @@
         props[this.selectedAs] = true;
         props.tabIndex = true;
         this._instanceProps = props;
-        this._userTemplate = Polymer.dom(this).querySelector('template');
+        this._userTemplate = this.queryEffectiveChildren('template');
```

Comment:
One last thing, in L922, the physical items are appended as direct children:
```js
Polymer.dom(this).appendChild(inst.root);
```
Let's change that to:
```js
Polymer.dom(this._userTemplate.parentNode).appendChild(inst.root);
```
That way, users can style the template from where the template is defined.